crates/rover-client/src/operations/subgraph/check_workflow/runner.rs

```diff
@@ -186,33 +198,42 @@ fn get_target_url_from_data(data: QueryResponseData) -> Option<String> {
 }
 
 fn get_check_response_from_result(
-    operations_result: Option<
+    maybe_operations_result: Option<
         SubgraphCheckWorkflowQueryGraphCheckWorkflowTasksOnOperationsCheckTaskResult,
     >,
-    operations_target_url: Option<String>,
+    display_target_url: Option<String>,
     number_of_checked_operations: u64,
     workflow_status: CheckWorkflowStatus,
     graph_ref: GraphRef,
     core_schema_modified: bool,
 ) -> Result<CheckResponse, RoverClientError> {
-    let result = operations_result.ok_or(RoverClientError::AdhocError {
-        msg: "Operations check task has no result.".to_string(),
-    })?;
-    let mut changes = Vec::with_capacity(result.changes.len());
-    for change in result.changes {
-        changes.push(SchemaChange {
-            code: change.code,
-            severity: change.severity.into(),
-            description: change.description,
-        });
-    }
+    match maybe_operations_result {
+        Some(result) => {
+            let mut changes = Vec::with_capacity(result.changes.len());
+            for change in result.changes {
+                changes.push(SchemaChange {
+                    code: change.code,
+                    severity: change.severity.into(),
+                    description: change.description,
+                });
+            }
 
-    CheckResponse::try_new(
-        operations_target_url,
-        number_of_checked_operations,
-        changes,
-        workflow_status.into(),
-        graph_ref,
-        core_schema_modified,
-    )
+            OperationCheckResponse::try_new(
+                display_target_url,
+                number_of_checked_operations,
+                changes,
+                workflow_status.into(),
+                graph_ref,
+                core_schema_modified,
+            )
+            .map(CheckResponse::OpeartionCheckResponse)
+        }
+        None => Ok(CheckResponse::OpeartionLessCheckResponse(
```

Comment:
```suggestion
        None => Ok(CheckResponse::SkipOperationsCheckResponse(
```

maybe a good name for this? happy to keep the current one too but should at least update to `Operation` instead of `Opeartion` - if you're using VS code and rust-analyzer, refactoring this is really easy, just press `F2` and it will rename all instances across the codebase using the type system.

Reply:
Renamed and fixed typos.